src/partitioning/centroid_partitioner.C

```diff
@@ -38,6 +38,10 @@ void CentroidPartitioner::partition_range(MeshBase & /*mesh*/,
       return;
     }
 
+  // We don't yet support distributed meshes with this Partitioner
+  if (!mesh.is_serial())
```

Comment:
Is `is_serial()` deprecated along with `SerialMesh`? Perhaps favor `is_replicated()`? Not a big deal, just noticed when I glanced to see what partioners don't support `DistributedMesh`

Reply:
Those aren't just a name change; they are two different methods!

is_serial()
```
   * \returns \p true if all elements and nodes of the mesh
   * exist on the current processor, \p false otherwise
```

whereas is_replicated()
```
   * \returns \p true if new elements and nodes can and should be
   * created in synchronization on all processors, \p false otherwise
```

In other words, DistributedMesh::is_serial() can return true!  And it's the right thing to use here: if e.g. a user wants to keep their DistributedMesh serialized long enough to get a space-filling-curve partitioning of the coarse mesh, but then hand off to ParMETIS for any subsequent repartitioning, there's no reason we can't or shouldn't let them.

Reply:
Thanks for the clarification! 1 more week until I can purge teaching for a month. I need to hide and code my ass off to get back in the swing. Why'd I take this job again?